#pragma once

namespace FU::Exception {

	//An exception thrown when a serious error occurs, 
	//not caught until the main function.
	class Serious final {};

}